<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>731</width>
    <height>663</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QWidget" name="horizontalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>420</x>
      <y>10</y>
      <width>301</width>
      <height>31</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="TimeLayout">
     <item>
      <widget class="QLabel" name="upTimeValue">
       <property name="text">
        <string>uptime</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="timeValue">
       <property name="text">
        <string>time</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="Line" name="line_2">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="batteryValue">
       <property name="lineWidth">
        <number>0</number>
       </property>
       <property name="text">
        <string>battery</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="batteryPercent">
       <property name="text">
        <string>%</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QWidget" name="horizontalLayoutWidget_2">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>110</y>
      <width>711</width>
      <height>334</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="BOX">
     <item>
      <layout class="QVBoxLayout" name="CpuRamLayout">
       <item>
        <layout class="QVBoxLayout" name="CPUMODULE">
         <item>
          <widget class="QLabel" name="CPUModuleLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>CPU</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring></cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="CpuSep">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_12">
           <item>
            <widget class="QLabel" name="label_11">
             <property name="text">
              <string>Temperature</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="cputemp">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="cpuInsert"/>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="RAMMODULE">
         <item>
          <widget class="QLabel" name="RAMModuleLabel">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>RAM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring></cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_3">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ramInsert_6">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_5">
             <item>
              <widget class="QLabel" name="maxramlabel_5">
               <property name="text">
                <string>Process</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="processVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_7">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ramInsert">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout">
             <item>
              <widget class="QLabel" name="maxramlabel">
               <property name="text">
                <string>Max Ram</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="maxramVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ramInsert_3">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_2">
             <item>
              <widget class="QLabel" name="maxramlabel_2">
               <property name="text">
                <string>Used Ram</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="usedramVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_2">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <widget class="Line" name="line_8">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ramInsert_4">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_3">
             <item>
              <widget class="QLabel" name="maxramlabel_3">
               <property name="text">
                <string>Max Swap</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="maxswapVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_3">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ramInsert_5">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_4">
             <item>
              <widget class="QLabel" name="maxramlabel_4">
               <property name="text">
                <string>Used Swap</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="usedswapVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_4">
               <property name="text">
                <string>Go</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_4">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="DISKNETW">
       <item>
        <layout class="QVBoxLayout" name="CPUMODULE_2">
         <item>
          <widget class="QLabel" name="CPUModuleLabel_2">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>DISK</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring></cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="CpuSep_2">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="cpuInsert_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_6">
             <item>
              <widget class="QLabel" name="label_5">
               <property name="text">
                <string>Read</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="readVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
           <item>
            <layout class="QVBoxLayout" name="cpuInsert_3">
             <item>
              <layout class="QHBoxLayout" name="horizontalLayout_7">
               <item>
                <widget class="QLabel" name="label_6">
                 <property name="text">
                  <string>Write</string>
                 </property>
                </widget>
               </item>
               <item>
                <widget class="QLabel" name="writeVal">
                 <property name="text">
                  <string>TextLabel</string>
                 </property>
                </widget>
               </item>
              </layout>
             </item>
            </layout>
           </item>
          </layout>
         </item>
        </layout>
       </item>
       <item>
        <spacer name="verticalSpacer_2">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeType">
          <enum>QSizePolicy::Fixed</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>0</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <layout class="QVBoxLayout" name="RAMMODULE_2">
         <item>
          <widget class="QLabel" name="RAMModuleLabel_2">
           <property name="font">
            <font>
             <pointsize>15</pointsize>
             <weight>75</weight>
             <bold>true</bold>
            </font>
           </property>
           <property name="text">
            <string>NETWORK</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
           <property name="buddy">
            <cstring></cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="Line" name="line_5">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QVBoxLayout" name="ramInsert_2">
           <item>
            <layout class="QHBoxLayout" name="horizontalLayout_8">
             <item>
              <widget class="QLabel" name="label_7">
               <property name="text">
                <string>Bytes Received</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="breceivedVal">
               <property name="text">
                <string>TextLabel</string>
               </property>
              </widget>
             </item>
             <item>
              <widget class="QLabel" name="label_12">
               <property name="text">
                <string>Mo/s</string>
               </property>
              </widget>
             </item>
            </layout>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_9">
           <item>
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Bytes Transmitted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="btransmittedVal">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="breceivedVal_2">
             <property name="text">
              <string>Mo/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_10">
           <item>
            <widget class="QLabel" name="label_9">
             <property name="text">
              <string>Packets Received</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="preceivedVal">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="breceivedVal_3">
             <property name="text">
              <string>Mo/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_11">
           <item>
            <widget class="QLabel" name="label_10">
             <property name="text">
              <string>Packets Transmitted</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="ptransmittedVal">
             <property name="text">
              <string>TextLabel</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="breceivedVal_4">
             <property name="text">
              <string>Mo/s</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <widget class="Line" name="line_6">
       <property name="orientation">
        <enum>Qt::Vertical</enum>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QVBoxLayout" name="USERMODULE">
       <item>
        <widget class="QLabel" name="DiskModuleLabel_2">
         <property name="font">
          <font>
           <pointsize>15</pointsize>
           <weight>75</weight>
           <bold>true</bold>
          </font>
         </property>
         <property name="text">
          <string>User</string>
         </property>
         <property name="alignment">
          <set>Qt::AlignCenter</set>
         </property>
         <property name="buddy">
          <cstring></cstring>
         </property>
        </widget>
       </item>
       <item>
        <widget class="Line" name="DiskSep_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
        </widget>
       </item>
       <item>
        <layout class="QHBoxLayout" name="userItemLayout">
         <item>
          <widget class="QLabel" name="kernel">
           <property name="text">
            <string>Kernel</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="kernelValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rowUserLayout2">
         <item>
          <widget class="QLabel" name="kernel_2">
           <property name="text">
            <string>OS Version</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="osversionValue">
           <property name="maximumSize">
            <size>
             <width>150</width>
             <height>150</height>
            </size>
           </property>
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rowUserLayout3">
         <item>
          <widget class="QLabel" name="machine">
           <property name="text">
            <string>Machine</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="machineValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="rowLayoutDisk_2">
         <item>
          <widget class="QLabel" name="sysName">
           <property name="text">
            <string>System Name</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="sysNameValue">
           <property name="text">
            <string/>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
    </layout>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>731</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
